packages/components/src/focusable-iframe/stories/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { text } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import FocusableIframe from '../';
+
+export default {
+	title: 'Components/FocusableIframe',
+	component: FocusableIframe,
+};
+
+export const _default = () => {
+	const source = text( 'iFrame Source', './' );
```

Comment:
how about using something else for the source?

Reply:
Any suggestions? I tried wordpress.org but it didn't work, I assumed because of security settings